gui/src/components/ProviderNetworks.tsx

```diff
@@ -62,10 +50,12 @@ export function ProviderNetworks({ children }: { children: ReactNode }) {
         id: `${actionId}/${network.name}`,
         name: network.name,
         parent: actionId,
-        perform: () => value.setCurrentNetwork(network.name),
+        perform: () => {
+          setCurrentNetwork(network.name);
+        },
       })),
     ],
-    [networks, value.setCurrentNetwork]
+    [networks, setCurrentNetwork]
```

Comment:
why is `setCurrentNetwork` still here if it's on `ProviderCurrentNetwork` as well?

Reply:
you need that for the kbar. But yeah, here is just a bug. the current network is yet to be instantiated at this point. Ill fix it